drivers/hmis/app/graphql/types/base_field.rb

```diff
@@ -40,5 +43,17 @@ def resolve(object:, arguments:, **_rest)
         Types::PaginatedScope.new(resolved_object, **pagination_arguments)
       end
     end
+
+    # Support setting a default value for a field
+    # Credit:  https://github.com/rmosolgo/graphql-ruby/issues/2783#issuecomment-592960093
+    class DefaultValueExtension < GraphQL::Schema::FieldExtension
+      def after_resolve(value:, **_rest)
+        if value.nil?
+          options[:default_value]
+        else
+          value
+        end
+      end
```

Comment:
Neat! I wonder if we should call super here to be safe. That way we bring in whatever [behavior is in the parent](https://github.com/rmosolgo/graphql-ruby/blob/b9524c292958efb567061ce04cc766f1a2a37fc4/lib/graphql/schema/field/scope_extension.rb#L6)
```suggestion
     def after_resolve(value:, **kwargs)
        value_with_default = value.nil? ? options[:default_value] : value
        super(value: value_with_default, **kwags)
     end
```

Reply:
I don't know though, the docs and examples for field extensions don't mention that you need to call super https://graphql-ruby.org/type_definitions/field_extensions.html
I think I'm going to leave it as is since I'm not sure what all the other code is doing